fe/fe-core/src/main/java/com/starrocks/sql/analyzer/DropStmtAnalyzer.java

```diff
@@ -52,6 +53,13 @@ public Void visitDropTableStmt(DropTableStmt statement, ConnectContext context)
                     } else {
                         ErrorReport.reportSemanticException(ErrorCode.ERR_BAD_TABLE_ERROR, tableName);
                     }
+                } else {
+                    if (table instanceof MaterializedView) {
+                        throw new SemanticException(
+                                "The data of '%s' cannot be dropped because '%s' is a materialized view," +
```

Comment:
The data of '%s' cannot be dropped because '%s' is a materialized view, you can use drop materialized to drop it.

Reply:
‘%s’ can not be dropped because it is a materialized view. use 'drop materialized view %s' to drop it.